Resume-6-12.md

```diff
@@ -0,0 +1,44 @@
+# Resume
+
+## Q6. Memory managed
+
+* The computer memory is like a empty book. Different writers want to write some story there but to do it they need to ask for permission to the book's manager because this show where they can write without over write the other story.
```

Comment:
```suggestion
* The computer memory is like a empty book. Different writers want to write a story, but to do it they need to ask for permission to the book's manager because only he can show where they can write without over write the other story.
```